src/js/components/AppVersionComponent.jsx

```diff
@@ -161,6 +161,11 @@ var AppVersionComponent = React.createClass({
         return <dd key={u}>{u}</dd>;
       });
 
+    var acceptedResourceRoles = (appVersion.acceptedResourceRoles == null ||
```

Comment:
I don't think we need the `null` check given https://github.com/mesosphere/marathon-ui/pull/222/files#diff-8b464c3ab39121e7485cfce09753880bR6